### ISSUE TITLE
Unify gradle version across sub-projects

### ISSUE THREAD

**Issue by akoehn** [dialogos-project/dialogos#176]
Our gradle versions are all over the place, as I saw when investigating #168.
 - ros, sqlite: 4.0.1
 - ev3, distribution, minimalio: 4.4.1
 - dialogos, alexa, jython: 4.10.2

I think we should have them all using the same version; probably update it to 5.2 (the current version) so we don't get weird inconsistencies when doing the same stuff in different sub-projects.

As for the milestone:  I would do that after 2.1.1

**Comment by alexanderkoller** [comment]
Sounds great.

One word of warning: Sometimes I get warnings that our buildfiles are using deprecated functionality that will no longer work in Gradle 5 (I think the names for dependency types have changed, like "compile" and "implementation" and such). If we upgrade to Gradle 5, we'll have to deal with that.

**Comment by akoehn** [comment]
Yes, that is why I would like to only do that after the next release.

It is probably a good idea to first try the build with 4.10, which
already give deprecation warnings.  Gradle has information regarding
updating to 5.x here:

https://docs.gradle.org/current/userguide/upgrading_version_4.html

**Comment by timobaumann** [comment]
for the releases, we're effectively using what's in distribution, right? So that's a single change that, if it does not yield errors, can be used across all sub-projects.

**Comment by akoehn** [comment]
> for the releases, we're effectively using what's in distribution,
> right?

No, the other artifacts are pre-built by jitpack using their respective
gradle version.

**Comment by timobaumann** [comment]
right.

**Comment by timobaumann** [comment]
ROS does not seem to like 5.2.1 but works fine with 4.10.2. The most recent 4-line version is 4.10.3 which I would propose we use.

**Comment by timobaumann** [comment]
see 4d4bee48027cf028f3f8ed81be3a66e4dd9437a6

**Comment by timobaumann** [comment]
I've done this for all repos that I feel (somewhat) responsible for. Can someone else do it for Lego and Alexa?
